node/cmd/main.go

```diff
@@ -25,6 +25,14 @@ import (
 	"github.com/ibm/ibm-block-csi-driver/node/pkg/driver"
 )
 
+func printVersion(configFile *string) {
+	info, err := driver.GetVersionJSON(*configFile)
+	if err != nil {
+		logger.Panicln(err)
```

Comment:
since the program continues normally, I think "error" is enough

Reply:
Done